test/functional/legacy/increment_spec.lua

```diff
@@ -720,4 +739,10 @@ describe('Ctrl-A/Ctrl-X on visual selections', function()
       eq({}, nvim.get_vvar('errors'))
     end)
   end
+
+  it('does not drop leading zeroes', function()
+    execute('set nrformats&vi') -- &vi makes Vim compatible
```

Comment:
It is in the original test, but there `set nrformats&vim` is called... but ok, if that's all  it does we can keep it :) But what's that syntax, can you explain that to me?